### ISSUE TITLE
Audit log / command list and time. 

### ISSUE THREAD

**Issue by pooley92** [NebraLtd/dashboard-issues#19]
On each hotspot it would be cool to have a timestamp of the last command sent to the device or when the request was sent. 

This will prevent me from forgetting and doing it multiple times.

**Comment by vpetersson** [comment]
When you say 'command', you mean actions such as reboot/restart/shut down?

**Comment by pooley92** [comment]
Yes this is correct. Just a simple date and time stamp of the request.
